### DISCUSSION TITLE
Upgrading from 8.0.0-R24021201 causes "Do not call GetAuthenticationStateAsync outside of the DI scope" error

### DISCUSSION THREAD

**Discussion by lwmorris067** [MarimerLLC/csla#3901] [Questions]
I've been struggling with this error. It occurs anytime the dataportal is called from the server or webassembly.

To reproduce:
- Load the blazor example
- Upgrade the nuget packages
- Comment out line 18 in the CslaStateController
- Comment out line 22 in home.razor
- Run the app.

Any help how to fix that error would be great! Until then, I've reverted my projects back to 8.0.0-R24021201.

**Comment by ossendorf-at-hoelscher** [comment]
Hi. Please see this issue for more information: https://github.com/MarimerLLC/csla/discussions/3815
